src/components/index.tsx

```diff
@@ -0,0 +1,9 @@
+export { default as DataViews } from './dataviews/DataViewTable';
+export { default as useWindowDimensions } from '@/hooks/ViewportDimensions';
```

Comment:
Pls export the hook from `index` of the *hooks* folder  so that we can use `dokan.hooks` and `dokan.components` separately.